### PR TITLE
Adds query limit

### PR DESCRIPTION
**Pull request by brunocalza**
## Summary

Adds query limit as discussed in [Limit query size to 35kb for all chains](https://www.notion.so/textile/Tableland-Strategy-Proposal-1f6a59949f914810850686ba24d0bf73)

### CODE REVIEW

File: contracts/TablelandTables.sol [hunk 1]
```diff
@@ -71,6 +73,10 @@ contract TablelandTables is
             revert Unauthorized();
         }
 
+        if (bytes(statement).length > QUERY_MAX_SIZE) {
+            revert("query size greater than maximum");
```
Comment (carsonfarmer) [inline contracts/TablelandTables.sol:77]:
```suggestion
            revert("Tables: query size greater than maximum");
```

File: test/TablelandTables.ts [hunk 2]
```diff
@@ -197,4 +197,23 @@ describe("TablelandTables", function () {
     tx = await tables.connect(owner).createTable(owner.address, createStatement)
     await tx.wait()
   })
+
+  it("Big query execution should fail", async function () {
```
Comment (carsonfarmer) [inline test/TablelandTables.ts:201]: Yes!

**Review by carsonfarmer — APPROVED** [review]

File: contracts/TablelandTables.sol [hunk 3]
```diff
@@ -26,6 +26,8 @@ contract TablelandTables is
     string private _baseURIString;
     // A mapping of table ids to table controller addresses.
     mapping(uint256 => address) private _controllers;
+    // The maximum size allowed for a query.
+    uint256 private constant QUERY_MAX_SIZE = 35000;
```
[the remark this answers is not in the document]
Comment (sanderpick) [reply contracts/TablelandTables.sol:30]: hmm, good point. i'm inclined to say that it's okay unless we want it to be different per network. looks like there is no problem with constants changing in upgradable contracts either.

File: contracts/TablelandTables.sol (hunk 1 above)
Comment (sanderpick) [reply contracts/TablelandTables.sol:77]: oh, actually we could create a custom error for this... see the `Unauthorized` error in the interface

**Review by sanderpick — APPROVED** [review]
sweet

**Pull request merged**